ghedesigner/search_routines.py

```diff
@@ -214,28 +214,29 @@ def search(self):
             pass
         else:
             # This domain does not bracket the solution
-            if t_0_upper < 0.0 and t_m1 < 0.0:
-                msg = (
-                    "Based on the loads provided, the excess temperatures \n"
-                    "for the minimum and maximum number of boreholes falls \n"
-                    'below 0. This means that the loads are "miniscule" or \n'
-                    "that the lower end of the domain needs to contain \n"
-                    "fewer boreholes."
-                )
-                raise ValueError(msg)
+            if t_0_upper < 0.0 and t_m1 < 0.0 and t_0_lower < 0.0:
```

Comment:
@TNWest is there any reason to check all three conditions? If the excess temperature is <0, it implies that the configuration is oversized. If we're checking the smallest configuration and it is <0, then presumably larger configurations would also have a negative excess temperature.

Reply:
Let me know if you agree with that logic. If so, I can make the edits.

Reply:
@mitchute You are correct; only one of the temperatures need to be checked in line 217 and 226 as they should only be executed when all three excess temperatures have the same sign. Even though redundant, I thought having all three temperatures could make the overall logic for the edge cases more clear, but if you disagree, I am not opposed to only checking "t_0_lower" for line 217 and "t_m1" for line 226.